src/libs/Navigation/Navigation.js

```diff
@@ -196,6 +202,19 @@ function isActiveRoute(routePath) {
     return getActiveRoute().substring(1) === routePath;
 }
 
+/**
+ * Navigate to the route that we originally intended to go to
+ * but the NavigationContainer was not ready when navigate() was called
+ */
+function goToPendingRoute() {
+    if (pendingRoute === null) {
```

Comment:
it would be (!pendingRoute) right?

Reply:
Yeah, in the original PR we had `pendingRoute === false` when `pendingRoute` is a string variable. In this case, I just changed it to `null` to make my vscode linter happy (I like to have a static strict mode in JS hehe). 

Said that I think Robert did it in that way to differentiate from empty strings. Seems this is to allow scenarios where `pendingRoute` could be the home route (empty string), otherwise, if we do `if (!pendingRoute)` I think we won't have the expected behavior for empty strings.

Reply:
Ooohhhh home route is empty string?? Ok I think you're right, let's keep it :D

Also @dangrous good point it would be `(!pendingRoute)` 😅 (darn logic flip-flopping)